crowdsec-docs/docs/appsec/configuration.md

```diff
@@ -4,74 +4,234 @@ title: AppSec Component Configuration Files
 sidebar_position: 6
 ---
 
-## Foreword
+## Overview
 
-Configuring the AppSec Component usually requires the use of multiple files:
+This section covers the detailed configuration options for the CrowdSec AppSec Component. 
 
- - [AppSec rules](/appsec/rules_syntax.md) allow you to write a signature to detect and/or block malevolent requests. [You can find more information about the syntax here](/appsec/rules_syntax.md)
- - [Acquisition configuration](/log_processor/data_sources/appsec.md) indicates which port is the AppSec Component listening on, and which AppSec configuration it will use.
- - AppSec configuration tells which rules are loaded in in-band (blocking) and out-of-band (non-blocking)
-  phases. [it as well allows you to tweak the behavior of the component via the powerful expr bindings](/appsec/rules_syntax.md)
+**Prerequisites**: 
+- Familiarity with [AppSec concepts](/appsec/intro.md)
+- Basic AppSec setup completed (see Getting Started guides)
 
-## Acquisition configuration
+The AppSec Component configuration consists of three main parts:
 
-## Default configuration
+ - **[Acquisition configuration](/log_processor/data_sources/appsec.md)**: Defines which port the AppSec Component listens on and which AppSec configurations to load <!-- Fix linked page to ie. speak about appsec_configs-->
+ - **AppSec configurations**: Define which rules are loaded and how they behave, along with [hooks](/appsec/hooks.md) for runtime customization
+ - **[AppSec rules](/appsec/rules_syntax.md)**: The actual detection signatures that identify and block malicious requests
 
-The Acquisition configuration is usually present directly within `/etc/crowdsec/acquis.d/` or `/etc/crowdsec/acquis.yaml`:
+## Acquisition Configuration
 
-> The default AppSec acquisition configuration
-```yaml
-appsec_config: crowdsecurity/appsec-default
+### Multiple AppSec Configurations
+
+Use `appsec_configs` to load multiple configurations that work together:
+
+```yaml title="/etc/crowdsec/acquis.d/appsec.yaml"
+appsec_configs:
+  - crowdsecurity/appsec-default    # In-band virtual patching
+  - crowdsecurity/crs               # Out-of-band detection based on ModSec CRS - from crowdsecurity/appsec-crs collection
 labels:
   type: appsec
 listen_addr: 127.0.0.1:7422
 source: appsec
 ```
 
-## Creating custom configuration
+:::info
+Do not forget to `sudo cscli collections install crowdsecurity/appsec-crs`.
+This collection installs OWASP CRS in out-of-band and adds a scenario to ban IPs triggering multiple rules.
+:::
 
+### Creating Custom Configurations
 
-If you want to add some custom rules or hooks, it is suggested to add a custom `appsec_config`.
-Modifying existing `appsec_config` will make it *tainted* and will interfere with future updates.
+Create new configuration files instead of modifying existing hub configurations. Modifying hub configurations will make them *tainted* and prevent automatic updates.
```

Comment:
I think here that talking about it within an acquisition example is confusing.

```suggestion
If you want to alter the default configuration files we recomend creating a new configuration files instead of modifying existing hub configurations. Modifying hub configurations will make them *tainted* and prevent automatic updates.
```